orchestrator/careplanservice/handle_createtask.go

```diff
@@ -96,19 +96,46 @@ func (s *Service) handleCreateTask(ctx context.Context, request FHIRHandlerReque
 		if err != nil {
 			return nil, fmt.Errorf("invalid Task.basedOn: %w", err)
 		}
-		// we have a valid reference to a CarePlan, use this to retrieve the CarePlan and CareTeam to validate the requester is a participant
+
 		var careTeams []fhir.CareTeam
 		err = s.fhirClient.Read(*carePlanRef, &carePlan, fhirclient.ResolveRef("careTeam", &careTeams))
 		if err != nil {
 			return nil, err
 		}
-		if len(careTeams) == 0 {
-			return nil, coolfhir.NewErrorWithCode("CareTeam not found in bundle", http.StatusNotFound)
-		}
 
-		participant := coolfhir.FindMatchingParticipantInCareTeam(careTeams, principal.Organization.Identifier)
-		if participant == nil {
-			return nil, coolfhir.NewErrorWithCode("requester is not part of CareTeam", http.StatusUnauthorized)
+		// Different validation logic for an SCP subtask
+		if coolfhir.IsScpTask(&task) && len(task.PartOf) > 0 {
```

Comment:
The `coolfhir.IsScpTask(&task)` part should also apply to the `else`-scope I would say. We probably want to throw an error when a non-SCP Task is sent to the CPS?

Reply:
Sounds good. I initially thpought we should support it since Task Filler skips it which made me think it was possible to create a non-SCP task but have it just not go through the task filler logic. I will add a check earlier in the code

Reply:
Yeah not a 100% sure, but it feels weird sending a non-SCP Task to the CPS. Maybe best to be restrictive for now, and we'll find out soon enough if it is required 😂